engine/h2shared/snd_sys.h

```diff
@@ -92,11 +93,13 @@
 
 #if defined(PLATFORM_AMIGA)
 #define HAVE_AHI_SOUND		1
+#define HAVE_PAULA_SOUND		1
```

Comment:
These should be guarded by `PLATFORM_AMIGAOS3`, yes?

Reply:
Yes: MOS or AROS linkage fails, otherwise.

Reply:
I forgot that PLATFORM_AMIGA is the common macro, I'll do that.